packages/chakra-ui-lib/src/components/Checkbox.tsx

```diff
@@ -143,6 +145,12 @@ export default implementRuntimeComponent({
       setChecked(!!defaultIsChecked);
     }, [setChecked, defaultIsChecked]);
 
+    useEffect(() => {
+      if (getElement && ref.current) {
+        getElement(ref.current.parentElement as HTMLElement);
```

Comment:
@tanbowensg 
Do we need to call `getElement` in every render call? This `useEffect` hook does not have a dependency array.

Reply:
Here is a mistake, it should be:
```javascript
    useEffect(() => {
      if (getElement && ref.current) {
        getElement(ref.current.parentElement as HTMLElement);
      }
    }, [getElement, ref]);
```

Reply:
I think the ref does not need to be added to the dependency.

https://epicreact.dev/why-you-shouldnt-put-refs-in-a-dependency-array/